next-is.js

```diff
@@ -522,6 +522,84 @@ is.js may be freely distributed under the MIT Licence.
           return !!ua.match(new RegExp(inAppBrowserDetectionRules , 'ig'));
         }
       }
+    },
+    isWindowsTabletAsync: function() {
+      // windows tablets are: Surface Pro, Lenovo Miix, HP Envy x2 etc
+      // we detect it based on:
+      // - has high pixel density
+      // - has front and back cameras
+      // - has touch
+      return new Promise(function(resolve, reject) {
+        if (
+          !is.windows()
+          || !is.userMediaSupported()
```

Comment:
What do you think about putting those `||`s at the ends of lines?

Reply:
It's personal preference to do it this way

Reply:
```
        if (
          !is.windows()
          || !is.userMediaSupported()
          || navigator.maxTouchPoints < 3 // less than 3 touch points - potato
          || root.devicePixelRatio < 1.5
        ) return reject(); 
```
vs
```
        if (
          !is.windows() ||
          !is.userMediaSupported() ||
          navigator.maxTouchPoints < 3 || // less than 3 touch points - potato
          root.devicePixelRatio < 1.5
        ) return reject(); 
```

Reply:
yeah, for me `||` in the beginning of the line looks better.
But yeah, It's a possible bug source in case doing that in `return` statement